src/download.ts

```diff
@@ -64,17 +63,14 @@ export type GetFileContentResponse<T = unknown> = {
 /**
  * The response for a get metadata request.
  *
- * @property contentType - The type of the content.
```

Comment:
So `contentType` will be dropped permanently right. Just checking whether this was an oversight or intentional. I guess you no longer have access to it through the headers returned by get metadata.

Reply:
Yep, and the content type is part of the metadata already